packages/typespec-vscode/src/extension.ts

```diff
@@ -276,3 +289,29 @@ function showStartUpMessages(stateManager: ExtensionStateManager) {
     stateManager.cleanUpStartUpMessage(workspaceFolder.uri.fsPath);
   });
 }
+
+async function shendEmitterTelemetryData(activityId: string) {
+  const uris = await TraverseTspConfigFileInWorkspace();
+  if (uris.length === 0) {
+    logger.debug(
+      "No tspconfig.yaml file found in workspace.Therefore, it is impossible to send telemetry data of emitter",
```

Comment:
feels not needed. let's limit the logging for telemetry related functions which may be triggered lots of times if not handled properly in the future.